packages/bolt-connection/test/bolt/request-message.test.js

```diff
@@ -429,4 +429,192 @@ describe('#unit RequestMessage', () => {
       })
     })
   })
+
+  describe('BoltV5.1', () => {
+    it('should create HELLO message with notification filters', () => {
+      const userAgent = 'my-driver/1.0.2'
+      const authToken = { username: 'neo4j', password: 'secret' }
+      const notificationFilters = ['*.*', 'WARNING.*']
+
+      const message = RequestMessage.hello5x1(authToken, { userAgent, notificationFilters })
+
+      expect(message.signature).toEqual(0x01)
+      expect(message.fields).toEqual([
+        { username: 'neo4j', password: 'secret' },
+        { user_agent: userAgent, notifications: notificationFilters }
+      ])
+      expect(message.toString()).toEqual(
+        `HELLO {...} {"user_agent":"${userAgent}","notifications":["*.*","WARNING.*"]}`
+      )
+    })
+
+    it('should create HELLO message without notification filters if it is not supplied or null', () => {
+      ;[null, undefined].forEach(notificationFilters => {
```

Comment:
This for disambiguate since we are not using `;` at the end of lines. So, whenever a statement start with a list, the linter/formatter just add the `;` to avoid have a wrong interpretation.

Reply:
tl,dr; The `;` is ending the last statement.

Reply:
Oh. So that it's not interpreted as indexing the result of the previous instruction :brain: 

Almost as if JS was made with semicolons at the end of statements in mind :stuck_out_tongue_winking_eye: 

Thanks for the explanation anyway :raised_hands: